apps/judicial-system/backend/src/app/modules/notification/test/createTestingNotificationModule.ts

```diff
@@ -36,7 +36,14 @@ export const createTestingNotificationModule = async () => {
     ],
     controllers: [NotificationController, InternalNotificationController],
     providers: [
-      CourtService,
+      {
+        provide: CourtService,
+        useValue: {
+          createDocument: jest.fn(async () => {
```

Comment:
Do we need the mock implementation here? Is `jest.fn()` not enough?

Reply:
You are right, since we are not promise chaining call to createDocument, we do not need the async mock.